frontend/src/users/components/navbar_component/Navbar.js

```diff
@@ -0,0 +1,54 @@
+import React, { useContext, useState } from 'react';
+import { Link } from 'react-router-dom';
+import "./navbar.css";
+import navItemsData from './NavItemsData';
+// import { Icon } from '@iconify/react';
+import Logo from '../../../Images/Logo.png'
+import { AuthContext } from '../../../context/auth-context';
+
+export default function Navbar({size}) {
+  const [isClicked,setClicked]=useState(false);
+  const data=useContext(AuthContext);
+
+  const removeButton=()=>{
+    const elements = document.getElementsByClassName("nav-bar-links");
+    [...elements].forEach(element => {element.classList.remove("nav-button")});
```

Comment:
Here, another solution you could use, instead of using document.getElementsBy... , is, you can make a state boolean variable, and set the class of the nav-bar-links to be 'nav-button' depending on the value of this boolean state (`className={booleanState && "nav-button"}`) . And then, when you want to remove the class, just change the value of the boolean.
Just saying.
Preferably try to not use document or window methods much in React.